static/src/components/PlayerDisplay.js

```diff
@@ -28,87 +29,62 @@ export class PlayerDisplay extends Component {
         this.state = {
             userID: null,
             displayID: null,
-            displayData: null,
-            fetch: true,
+            userData: null,
         };
-        if (this.props.playerId) {
-            this.state.displayID = this.props.playerId;
-            this.state.userID = this.props.playerId;
+        if (this.props.game) {
```

Comment:
I would recommend always binding the methods because otherwise if this is null and there is another null reference inside the render method there will be an infinite render loop, whereas if these are bound the method will just throw an error